configuration: Add option to sync time with user

punfil opened the pull request:
PR has been tested with latest version of APK on Android Studio emulator.

punfil committed:
configuration: Add option to sync time with user

Signed-off-by: punfil <wojtek.panfil@hotmail.com>

punfil committed:
grafana: Adjust %H to %

In the second chart %H was still the unit, which was a gap.

Signed-off-by: punfil <wojtek.panfil@hotmail.com>

iwanicki92 commented on html/js/common.js:
```diff
@@ -5,4 +5,20 @@ function redirect(url, http){
+function sendUserTime(){
+  var userDate = new Date();
+  var moth = userDate.getUTCMonth();
+  var userTime = userDate.getFullYear() + "-" + String(userDate.getMonth()+1) + "-" + userDate.getDate() + " " + userDate.getHours() + ":" + userDate.getMinutes() + ":" + userDate.getSeconds();
```
Currently we use UTC timezone(local and universal) so you need to get UTC time e.g.
`userDate.getUTCFullYear();
userDate.getUTCHours();
`
Just add UTC after get.
And delete unused `var moth`

If Wiktor changes local timezone in system then it would still be preferable to convert to local timezone, maybe in python from UTC to local or we could just leave it so it only works in one timezone.

punfil replied:
@iwanicki92 Or maybe we shall set timezone as well?

punfil replied:
`var moth` was used for debug, good point, deleted.

Instead of setting UTC time (which is not on API of timedatectl?) first set timezone, then set local time, should work the same.

iwanicki92 replied:
If Wiktor adds timezones then ok, if not then just get UTC time.

punfil replied:
Auto timezone set added.

iwanicki92 reviewed (changes requested):
Please set UTC date

punfil committed:
grafana: Treat pressure as int not float

Sensor serves int value.

Signed-off-by: punfil <wojtek.panfil@hotmail.com>

punfil committed:
Disable dots on charts

Signed-off-by: punfil <wojtek.panfil@hotmail.com>

punfil committed:
grafana: Add aggregateWindow() to query

This allows to create avg value of time period so that constant number of points is received from database. It's not important for small data received, but resolves the issue when count(points) > max_data_size. Resolution is done according to grafana error hint

Signed-off-by: punfil <wojtek.panfil@hotmail.com>

esesyn01 reviewed (approved):
It works now good
